<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UISettingsFileHandler</class>
 <widget class="QWidget" name="UISettingsFileHandler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>ファイルハンドラー</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ComboBoxFileHandler" name="comboBox_text"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>*.bsa;*.ba2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ComboBoxFileHandler" name="comboBox_bsa"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>*.nif</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ComboBoxFileHandler" name="comboBox_nif"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>*.esm:*.esl;*.esp</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ComboBoxFileHandler" name="comboBox_esp"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBoxFileHandler</class>
   <extends>QComboBox</extends>
   <header>ComboBoxFileHandler.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
